test/e2e/storage/vsphere_volume_vsan_policy.go

```diff
@@ -207,10 +203,9 @@ var _ = SIGDescribe("Storage Policy Based Volume Provisioning [Feature:vsphere]"
 	})
 
 	It("verify an existing and compatible SPBM policy is honored for dynamically provisioned pvc using storageclass", func() {
-		By(fmt.Sprintf("Invoking Test for SPBM policy: %s", os.Getenv("VSPHERE_SPBM_GOLD_POLICY")))
-		goldPolicy := os.Getenv("VSPHERE_SPBM_GOLD_POLICY")
-		Expect(goldPolicy).NotTo(BeEmpty())
-		scParameters[SpbmStoragePolicy] = goldPolicy
+		By(fmt.Sprintf("Invoking Test for SPBM policy: %s",))
```

Comment:
By(fmt.Sprintf("Invoking Test for SPBM policy: %s",)) => Need to pass policy name here for console printing.